modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -0,0 +1,246 @@
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Luis Miguel Barcos
+ */
+@RunWith(Arquillian.class)
+public class ObjectEntryResourceTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	public void setUp() throws Exception {
+		_objectDefinition = _publishObjectDefinition(
+			Collections.singletonList(
+				ObjectFieldUtil.createObjectField(
+					"Text", "String", true, true, null,
+					RandomTestUtil.randomString(), _OBJECT_FIELD_NAME, false)));
+
+		_objectEntry = _addObjectEntry(_OBJECT_FIELD_VALUE);
+	}
+
+	@Test
+	public void testGetNestedFieldDetailsInOneToManyRelationship()
+		throws Exception {
+
+		String relatedObjectFieldName = "x" + RandomTestUtil.randomString();
+
+		ObjectDefinition relatedObjectDefinition = _publishObjectDefinition(
+			Collections.singletonList(
+				ObjectFieldUtil.createObjectField(
+					"Text", "String", true, true, null,
+					RandomTestUtil.randomString(), relatedObjectFieldName,
+					false)));
+
+		ObjectRelationship objectRelationship =
+			ObjectRelationshipLocalServiceUtil.addObjectRelationship(
+				TestPropsValues.getUserId(),
+				_objectDefinition.getObjectDefinitionId(),
+				relatedObjectDefinition.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				StringUtil.randomId(),
+				ObjectRelationshipConstants.TYPE_ONE_TO_MANY);
+
+		String relatedObjectFieldValue = "x" + RandomTestUtil.randomString();
+
+		ObjectEntry relatedObjectEntry =
+			ObjectEntryLocalServiceUtil.addObjectEntry(
+				TestPropsValues.getUserId(), 0,
+				relatedObjectDefinition.getObjectDefinitionId(),
+				HashMapBuilder.<String, Serializable>put(
+					relatedObjectFieldName, relatedObjectFieldValue
+				).build(),
+				ServiceContextTestUtil.getServiceContext());
+
+		User user = TestPropsValues.getUser();
+
+		ObjectRelationshipLocalServiceUtil.
+			addObjectRelationshipMappingTableValues(
+				user.getUserId(), objectRelationship.getObjectRelationshipId(),
+				_objectEntry.getPrimaryKey(),
+				relatedObjectEntry.getPrimaryKey(),
+				ServiceContextTestUtil.getServiceContext());
+
+		JSONObject jsonObject = _invoke(
+			StringBundler.concat(
+				relatedObjectDefinition.getRESTContextPath(),
+				"?nestedFields=r_", objectRelationship.getName(), "_",
+				_objectDefinition.getPKObjectFieldName()),
+			Http.Method.GET);
+
+		JSONArray itemsJSONArray = jsonObject.getJSONArray("items");
+
+		Assert.assertEquals(1, itemsJSONArray.length());
+
+		JSONObject itemJSONObject = itemsJSONArray.getJSONObject(0);
+
+		Assert.assertEquals(
+			relatedObjectFieldValue,
+			itemJSONObject.getString(relatedObjectFieldName));
+
+		_invoke(
```

Comment:
I don't know if this invocation makes sense as you are not getting the result of the GET request